packages/forms/resources/js/components/textarea.js

```diff
@@ -10,5 +10,22 @@ export default function textareaFormComponent({ initialHeight }) {
                 this.$el.style.height = this.$el.scrollHeight + 'px'
             }
         },
+
+        input: {
+            ['@intersect.once']() {
+                this.render()
+            },
+            ['@input']() {
+                this.render()
+            },
+            ['@resize.window']() {
+                this.render()
+            },
+            ['@form-validation-error.window']() {
+                this.$nextTick(() => {
+                    this.render()
+                })
+            },
+        },
```

Comment:
What's the reason for moving these into the JS file? 🙂

Reply:
it is not necessarily moving these in to js file indeed, but as per `resources/js/components/tags-input.js` using x-bind property, for me it is a best practice to implement that way :)